packages/clippy/src/ClippyProvider.jsx

```diff
@@ -6,6 +6,8 @@ import clippyStyle from './style';
 import AGENTS from './agents';
 import ClippyContext from './ClippyContext';
 
+if (window) window.CLIPPY_CDN = '//s3.amazonaws.com/clippy.js/Agents/';
+
```

Comment:
```suggestion
if (typeof window !== 'undefined') window.CLIPPY_CDN = '//s3.amazonaws.com/clippy.js/Agents/';

```

On server side, `window` will be `undefined` and will make this a false positive. We need to ensure that window isn't `undefined`

Reply:
if window is undefined javascript will treat it as a falsy value but I do agree the condition should be more explicit specially for readability. I'll make an update tonight